data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,288 @@ mission "Arachi Cartoonist 2"
+					goto nwp
+			
+			label "seen kimek, arachi"
+			choice
+				`	(Take the train to the Saryd game.)`
+					goto saryd
+				`	(Take the train to the cooperative game.)`
+					goto wp
+			
+			label "seen all"
+			choice
+				`	(Take the last train.)`
+					goto wp
+				
+			
+			label saryd
+			apply
+				set "coalition games: saryd"
+			`	You arrive at a massive, stadium-like structure, only it's actually shaped like a half moon. The place overlooks a huge valley. Many small drones fly over it, transmitting live footage to the dozens of cinema-sized screens for those that can't get a clear view through the glass.`
+			`	You notice starting gates on one of the sides of the track. A few minutes pass before an alarm sounds, and the massive crowd roars. The doors open: several Saryd burst from the gates with wheeled machines, similar to chariots, attached to them. They are also carrying bows and arrows.`
+			`	The contestants scatter among a variety of paths leading into the increasingly dense forest. Targets appear for a few seconds as the Saryds approach, and they attempt to shoot at them. Eventually, you have to turn and look at the massive monitors, as it becomes impossible to follow the race by looking out the window.`
+			`	One participant takes a corner going too fast, and their "chariot" slams into a tree. He slows to a stop and throws his bow into the ground. The drones don't transmit sound, but it's clear from his reaction he's been disqualified. Over the din of the crowd, you hear voices coming through the speakers, likely commentators.`
+			`	As they move deeper, tall brambles and ledges become more common among the path, and the targets appear faster and faster. A Saryd navigates one section perfectly, nailing seven different targets in a row. You notice his picture on another monitor rises several positions, putting him near the front. It seems that hitting more targets counts towards a point system.`
+			`	You weren't able to count how many started, but as the race approaches its end, you're able to see roughly thirty made it through - not many. As the Saryds cross a finish line marked by a stream, the monitors switch to a ground crew interviewing the finishers.`
+			`	It looks like Ulmodi finished fourth; he's up next to be interviewed. The event seems to be over, and the crowd disperses under the supervision of Heliarch agents. Instead of heading back to the train, they're moving to a different section of the oddly-shaped stadium.`
+			`	You go with the crowd and are shown to a room with many elevators. Eventually, everyone descends to the valley below and follows a path to an actual, full-size stadium, and a massive one at that.`
+			`	As you sit among the alien beings feeling a little out of place, you realize you're probably the first human ever to witness these games. You look over the stadium grounds. It's rectangular, and small mounds dot the field. Six large holes are arranged evenly on the sides of the field.`
```

Comment:
I'd move this bit about being the first human to witness these games to the line talking about waiting, and move the description of the stadium grounds to when you first enter. (In my suggestions I have moved the second sentence to the one above this and the first sentence to the one below this.) This way we flow from entering and describing the stadium to pondering on the player's place in all this and then moving on to what's happening with the next event, instead of entering the stadium, thinking about the player's place, *then* describing the stadium, then moving on to the event.
```suggestion
```